Dynavity/Dynavity/model/canvas/elements/CodeElement.swift

```diff
@@ -1,28 +1,21 @@
+import Combine
 import CoreGraphics
 import Foundation
 
-struct CodeElement: TextElementProtocol {
+class CodeElement: ObservableObject, CanvasElementProtocol, TextElementProtocol {
```

Comment:
Should we inherit `PlainTextElement` instead? Then we should be able to get rid of `TextElementProtocol`.

Same goes for `MarkupElement`.

Reply:
Can we make auxiliary changes in separate PRs? This one is already quite a nightmare. D:

Reply:
ok!